source/_components/watson_iot.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: "IBM Watson IoT Platform"
+description: "Record events in the IBM Watson IoT Platform."
+date: 2018-04-03 22:09
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ibm.png
+ha_category: History
+ha_release: 0.72
+---
+
+The `watson_iot` component enables you to link the devices in Home Assistant
+with an [IBM Watson IoT Platform instance](https://www.ibm.com/us-en/marketplace/internet-of-things-cloud).
+
+## {% linkable_title Configuration %}
+
+To use this component, you first need to register a gateway device type and then
+a gateway device in your IoT platform instance. For instructions on how to do
+this check the [official documentation](https://console.bluemix.net/docs/services/IoT/gateways/dashboard.html#IoT_connectGateway)
```

Comment:
Could I suggest adding a line about why you would use this component? "i.e. you can use this to bring IoT history data into Watson to build an application..." My thought is that people will see Watson and think they will get some out of the box analytics or machine learning.